frontend/src/components/common/form/DisplayError.tsx

```diff
@@ -5,15 +5,18 @@ import { useFormikContext, getIn } from 'formik';
 type DisplayErrorProps = {
   /** The form field to show errors for */
   field?: string;
+  /** Display error even if field hasn't been touched */
+  errorPrompt?: boolean;
 };
 
 /**
  * Formik-connected, thin wrapper around React-Bootstrap to display form validation errors
  */
-export const DisplayError: React.FC<DisplayErrorProps> = ({ field }) => {
-  const { errors } = useFormikContext();
+export const DisplayError: React.FC<DisplayErrorProps> = ({ field, errorPrompt }) => {
```

Comment:
This was technically a bug, the error shouldn't display unless the field is touched. Added a prop to give backwards compatibility.